tests/gdx-tests/src/com/badlogic/gdx/tests/g3d/shadows/utils/NearFarAnalyzer.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.badlogic.gdx.tests.g3d.shadows.utils;
+
+import com.badlogic.gdx.graphics.Camera;
+import com.badlogic.gdx.graphics.g3d.environment.BaseLight;
+import com.badlogic.gdx.math.Vector2;
+
+/** Nearfar Analyzer compute the near and far plane of a camera
+ * @author realitix */
+public interface NearFarAnalyzer {
+	/** Analyze near and far plane for the camera
+	 * @param light Current light
+	 * @param camera Light's camera
+	 * @return Vector2 x=near, y=far */
```

Comment:
Probably better to have it set the camera's near and far values instead of "abusing" a vector for that. Otherwise, where does this vector come from, who creates it (does it create garbage or is it reused e.g.)?